addons/binding/org.openhab.binding.zoneminder/ESH-INF/thing/server-bridge.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="zoneminder"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 org.eclipse.smarthome.thing-description.xsd">
+
+    <bridge-type id="server">
+        <label>ZoneMinder Server</label>
+        <description>ZoneMinder Server</description>
+        <channels>
+            <channel id="is-alive" typeId="server_is_alive">
+                <label>Is Alive</label>
```

Comment:
why do you overwrite label/description here? Better remove it here and set the proper texts in the channel type below.